tsdef/VideoProcessor.d.ts

```diff
@@ -1,6 +1,33 @@
+/**
+ * @beta VideoFrame is part of Video Processor V3 support which is currently in beta.
+ */
+declare class VideoFrame {
+  displayHeight: number;
+  displayWidth: number;
+  timestamp: number;
+  constructor(videoSource: HTMLVideoElement);
+  close(): void;
+}
+
+/**
+ * A processor for manipulating video frames.
+ * @beta Video Processor V3 support is currently in beta.
```

Comment:
Remove beta tag

Reply:
These APIs are used by other developers apart from Video Processors SDK, so let's remove references to the video processors sdk.

Reply:
Done 👍